packages/sdk-core/src/Operation.ts

```diff
@@ -36,13 +36,16 @@ export default class Operation {
      * Executes the operation via the provided signer.
      * @description Populates all fields of the transaction, signs it and sends it to the network.
      * @param signer The signer of the transaction
+     * @param gasLimitMultiplier A multiplier to provide gasLimit buffer on top of the estimated gas limit (1.2x is the default)
      * @returns {ethers.providers.TransactionResponse} A TransactionResponse object which can be awaited
      */
     exec = async (
-        signer: ethers.Signer
+        signer: ethers.Signer,
+        gasLimitMultiplier = 1.2
```

Comment:
From looking at about 10 transactions with no multiplier, the consumed gas on Etherscan never went over 90%. So I guess now it will start to hover around 70%-75%.

Reply:
This 1.2 is a buffer to account for unpredictability. In probably 99.9% of cases, this should result in gas being refunded at the end of the txn. Also pls re-review changes here.